AbstractSQLRules2SQL.ometajs

```diff
@@ -743,6 +744,23 @@ export ometa AbstractSQLRules2SQL {
 			)
 		),
 
+	Case :indent =
+		[
+			'Case'
+			[	When(indent)+:whens
```

Comment:
I think this would be nicer as
```
'Case'
When(indent)+:whens
AnyValue(indent):val
```
so you can do
```
[	'Case',
	[ 'When', ... ]
	[ 'When', ... ]
	[ 'When', ... ]
	[ 'When', ... ]
	[ ... ]
]
```
instead, it just makes it a slightly nicer format imo

Reply:
It's probably also worth making the `ELSE` explicit via `[ 'Else', ... ]`

Reply:
Agreed on both parts.